mod/05_email/set/type/email_template.rb

```diff
@@ -44,7 +44,7 @@ def email_config args={}
   
   [:subject, :text_message].each do |field_name|
     config[field_name] = process_email_field( field_name, args ) do |field_card|
-      field_card.contextual_content context_card, {:format=>'email_text'}, args
+      field_card.contextual_content context_card, {:format=>'email_text'}, args.merge(:content_opts=>{ :chunk_list=>:inclusion_and_link})
```

Comment:
Yes, for example "wagn.org" became "wagn.org[wagn.org]". Spam filters don't like that. I guess Ethan introduced this when he overwrote the link_to method (https://github.com/wagn/wagn/blame/master/mod/05_standard/set/all/links.rb).  
The format doesn't deal with this because chunk_list is only defined on sets and not on formats.
But maybe I should create a right/subject.rb and right/text_message to handle this more genereally.